src/system/drivers/event_bus_driver/support/event_profile_modified.rs

```diff
@@ -0,0 +1,85 @@
+use crate::prelude::*;
+
+/// The active profile has been modified (might not have been saved yet).
+#[derive(Debug, Clone, PartialEq, Eq, Hash, uniffi::Enum)]
+pub enum EventProfileModified {
+    /// A new account with `address` was inserted into the active profile
+    AddedAccount { address: AccountAddress },
+
+    /// New accounts with `addresses` were inserted into the active profile
+    AddedAccounts { addresses: Vec<AccountAddress> },
+
+    /// An existing account has been updated
+    UpdatedAccount { address: AccountAddress },
```

Comment:
I would rename to `AccountAdded`, `AccountsAdded` and `AccountUpdated`, since that's the syntax ned followed in `Event` and `EventKind`.